.sanitizerconfig

```diff
@@ -0,0 +1,417 @@
+    id: null
+    in_effect: null
+    municipality_id: null
+    origin_id: null
+  munigeo_poi:
+    category_id: null
+    description: null
+    id: null
+    location: null
+    municipality_id: null
+    name: null
+    origin_id: null
+    street_address: null
+    zip_code: null
+  munigeo_poicategory:
+    description: null
+    id: null
+    type: null
+  munigeo_street:
+    id: null
+    modified_at: null
+    municipality_id: null
+  munigeo_street_translation:
+    id: null
+    language_code: null
+    master_id: null
+    name: null
+  profiles_address:
+    address: "profile.street_address"
+    address_type: null
+    city: "profile.city"
+    country_code: "profile.country_code"
+    id: null
+    postal_code: "profile.postal_code"
+    primary: null
+    profile_id: null
+  profiles_claimtoken:
+    expires_at: null
+    id: null
+    profile_id: null
+    token: null
+  profiles_divisionofinterest:
+    division_id: null
+    id: null
+  profiles_email:
+    email: "profile.email"
+    email_type: null
+    id: null
+    primary: null
+    profile_id: null
+    verified: null
+  profiles_legalrelationship:
+    confirmation_degree: null
+    expiration: null
+    id: null
+    representative_id: null
+    representee_id: null
+    type: null
+  profiles_phone:
+    id: null
+    phone: "profile.phone"
+    phone_type: null
+    primary: null
+    profile_id: null
+  profiles_profile:
+    contact_method: null
+    first_name: "profile.first_name"
+    id: null
+    image: "constant.null"
+    language: null
+    last_name: "profile.last_name"
+    nickname: "string.empty"
+    user_id: null
+  profiles_profile_concepts_of_interest:
+    concept_id: null
+    id: null
+    profile_id: null
+  profiles_profile_divisions_of_interest:
+    administrativedivision_id: null
+    id: null
+    profile_id: null
+  profiles_sensitivedata:
+    id: null
+    profile_id: null
+    ssn: "profile.encrypted_national_identification_number"
+  profiles_temporaryreadaccesstoken:
+    created_at: null
+    id: null
+    profile_id: null
+    token: null
+    validity_duration: null
+  profiles_verifiedpersonalinformation:
+    _national_identification_number_data: "profile.encrypted_national_identification_number"
+    email: "profile.encrypted_email"
+    first_name: "profile.first_name"
+    given_name: "profile.encrypted_first_name"
+    id: null
+    last_name: "profile.last_name"
+    municipality_of_residence: "profile.encrypted_city"
+    municipality_of_residence_number: "profile.encrypted_city_number"
+    national_identification_number: "string.empty"
+    profile_id: null
+  profiles_verifiedpersonalinformationpermanentaddress:
+    id: null
+    post_office: "profile.encrypted_city"
+    postal_code: "profile.encrypted_postal_code"
+    street_address: "profile.encrypted_street_address"
+    verified_personal_information_id: null
+  profiles_verifiedpersonalinformationpermanentforeignaddress:
+    additional_address: null
```

Comment:
`profiles_verifiedpersonalinformationpermanentforeignaddress.additional_address` should probably be sanitised?